GOVERNANCE.md

```diff
@@ -72,8 +72,10 @@ membership beyond these rules.
 
 The CTC may add additional members to the CTC by a standard CTC motion.
 
-A CTC member may be removed from the CTC by voluntary resignation, or by
-a standard CTC motion.
+When a CTC member's participation in CTC matters has become minimal for a
```

Comment:
What is meant by "CTC matters"? Is that attending CTC meetings, reviewing code, contributing changes, helping others in nodejs/node issues/PRs, etc.? Is not doing one of these "CTC matters" enough to be "respectfully asked to resign"?

What is considered "minimal" and "a sustained period of time"?

In other words, this seems a bit too vague.

It also seems to be in contradiction with this line above:

```
There is no specific set of requirements or qualifications for CTC membership beyond these rules
```